templates/vhost/_file_header.erb

```diff
@@ -3,7 +3,7 @@
 # Managed by Puppet
 # ************************************
 
-<VirtualHost <%= @nvh_addr_port %>>
+<VirtualHost <%= [@nvh_addr_port].flatten.join(' ') %>>
```

Comment:
I'm not sure if it is even possible that there is a nil in that array but it won't break something...